dbt/adapters/databricks/connections.py

```diff
@@ -58,6 +61,26 @@
 
 logger = AdapterLogger("Databricks")
 
+
+class DbtCoreHandler(logging.Handler):
+    def __init__(self, level: Union[str, int], *args: Tuple, **kwargs: Mapping):
+        super().__init__(level=level)
+        self.logger = AdapterLogger("databricks-sql-connector")
+
+    def emit(self, record: logging.LogRecord) -> None:
+        # record.levelname will be debug, info, warning, error, or critical
+        # these map 1-to-1 with methods of the AdapterLogger
+        log_func = getattr(self.logger, record.levelname.lower())
+        log_func(record.msg)
+
+
+pysql_handler = DbtCoreHandler(level=0)
+
+pysql_logger = logging.getLogger("databricks.sql")
```

Comment:
don't we want this to be configured by an environment variable? Are you sure we always want to log debug level messages from sql connector?

Reply:
We can make it configurable. I believe the default for `dbt.log` is DEBUG which is why I kept it that way here.